src/internal/operators/switchAll.ts

```diff
@@ -1,10 +1,75 @@
-import { OperatorFunction, ObservableInput } from '../types';
+import {OperatorFunction, ObservableInput, MonoTypeOperatorFunction} from '../types';
 import { switchMap } from './switchMap';
 import { identity } from '../util/identity';
 
 export function switchAll<T>(): OperatorFunction<ObservableInput<T>, T>;
 export function switchAll<R>(): OperatorFunction<any, R>;
 
+/**
+ * Converts a higher-order Observable into a first-order Observable
+ * producing values only from the most recent observable sequence
+ *
+ * <span class="informal">Flattens an Observable-of-Observables.</span>
+ *
+ * ![](switchAll.png)
+ *
+ * `switchAll` subscribes to an Observable that emits Observables, also known as a higher-order Observable.
+ * The observable sequence that at any point in time produces the elements of the most recent inner
+ * observable sequence that has been received. The output Observable only completes once the source observable,
+ * and all inner Observables have completed. Any error delivered by the source or inner Observables will be immediately
+ * emitted on the output Observable.
+ *
+ * ## Examples
+ * Spawn a new interval Observable for each click event, but for every new
+ * click, it just cancel the previous one then it subscribes to the new one
+ * ```ts
+ * import { fromEvent } from "rxjs";
+ * import { switchAll, map } from "rxjs/operators";
+ *
+ * const clicks = fromEvent(document, 'click');
+ * const higherOrder = clicks.pipe(map((ev) => interval(1000)));
+ * const firstOrder = higherOrder.pipe(switchAll());
+ * firstOrder.subscribe(x => console.log(x));
+ * ```
+ *
+ * Hit on the screen and it will let you track the last 10 mouse position.
```

Comment:
This would be a good candidate to have a stackblitz example to go along with.

Reply:
This already have a stackbliz example but since we do not have support from it (stackbliz) I could not added it. https://stackblitz.com/edit/switchall-rxjs